### PR TITLE
CO: newbie post for Jason and Kip

### PR DESCRIPTION
**Pull request by COpenda**

### CODE REVIEW

**Comment by rowo** [comment]
@COpenda there are a few capitalization errors. What is "C-"? Is "App Work" a company? It's "mSpray". For a photo you could have J with a tennis ball and K with a soccer ball. Or just looking professional. I'll wait on that before I merge.

**Comment by COpenda** [comment]
@rowo It's actually Upwork. Ok, I'll add the pics once i get them.

**Comment by COpenda** [comment]
img_20160726_102728
Kip 
img_20150809_161536
img_20150816_153916
Two above for Jason.

File: _posts/2016-07-25-Welcome-Jason-and-Kip.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+type: post
+published: true
+---
+Last week saw two new members join our cause...
+
+Introducing Jason Rogena and Kipsigei Cheroigin!
+
+Jason has joined us as a software developer for Android. He previously worked as a developer for Sendy where he managed the servers and the back end code. At Ona, he is serving in the OpenSRP team. His immediate task is to make modifications to Open Street Maps to make it work well with mSpray. mSpray is a tool that uses ODK collect and street maps to keep track of malaria spray points. If he is not trying to figure out something ground breaking on Android, you’ll find him practising tennis with Serena Williams. Jason has a degree in computer science from the University of Nairobi.
+
+Kipsigei has joined us as a software development intern on Zebra specializing in both C# and Java. Before Ona, Kip was a freelance application developer for Upwork. Kip is an avid soccer fan and liverpool supporter who thinks Game of Thrones is overrated. In his free time, you’ll find him catching up on some of his favorite TV shows or enjoying a good soccer match. Kipsigei has a degree in computer technology from Jomo Kenyatta University.
```
Comment (urbanslug) [inline _posts/2016-07-25-Welcome-Jason-and-Kip.md]: LMAO at the GoT hate!

**Comment by rowo** [comment]
@COpenda There were still a few mistakes I had to fix. Liverpool is a place or team, either way it's capitalized.  "...modificationsOpenMapKit". IRS **s**pray lowercase. "...programs across Southern Africa, this is a very..." isn't a proper transition. Groundbreaking as used is one word.

Not a mistake, but generally Zebra is an internal term. For external communications, Zebra is generally the user interface or front end or I just wrote "Ona data platform."

**Pull request merged**